sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusOptions.cs

```diff
@@ -193,19 +193,19 @@ internal Task ExceptionReceivedHandler(ProcessErrorEventArgs args)
             return Task.CompletedTask;
         }
 
-        internal ServiceBusProcessorOptions ToProcessorOptions() =>
+        internal ServiceBusProcessorOptions ToProcessorOptions(bool autoCompleteMessages) =>
             new ServiceBusProcessorOptions
             {
-                AutoCompleteMessages = AutoCompleteMessages,
+                AutoCompleteMessages = autoCompleteMessages,
                 PrefetchCount = PrefetchCount,
                 MaxAutoLockRenewalDuration = MaxAutoLockRenewalDuration,
                 MaxConcurrentCalls = MaxConcurrentCalls
             };
 
-        internal ServiceBusSessionProcessorOptions ToSessionProcessorOptions() =>
+        internal ServiceBusSessionProcessorOptions ToSessionProcessorOptions(bool autoCompleteMessages) =>
```

Comment:
It might be better to name this param something like `computedAutocompleteValue` or `autoCompleteMessagesOptionEvaluatedValue` like you have in the listener.

Reply:
Sure makes sense